cmake/Modules/FindORC.cmake

```diff
@@ -22,6 +22,10 @@ FIND_LIBRARY(ORC_LIB orc-0.4
              HINTS ${PC_ORC_LIBRARY_DIRS}
 	     PATHS ${ORC_ROOT}/lib${LIB_SUFFIX} ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
+FIND_LIBRARY(ORC_LIBRARY_STATIC liborc-0.4.a
+             HINTS ${PC_ORC_LIBRARY_DIRS}
+	     PATHS ${ORC_ROOT}/lib${LIB_SUFFIX} ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
```

Comment:
I mainly stumbled over this section before I added my "please format" comment.

I actually found a nice tool to do auto formatting: https://github.com/cheshirekow/cmake_format
But that would change a lot of lines.

I just open a new issue to propose cmake formatting and we can leave this as it is for the moment.